Statistics/Distributions/Empirical.cs

```diff
@@ -7,7 +7,9 @@
 using Utilities;
 
 namespace Statistics.Distributions
-{
+{/// <summary>
+/// TODO: we're not using this class. do we need to keep it??
```

Comment:
if there are zero references i would advise removing it. Be careful though, it might be hard to be certain it is not being used.